### PR TITLE
[MM-50081] Add work template Onboarding list entry

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Add an entry to the onboarding list for work templates.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-50081

#### Related Pull Requests
N/A

#### Screenshots
image

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: actions/views/onboarding_tasks.ts [hunk 1]
```diff
@@ -45,6 +47,21 @@ export function openInvitationsModal(timeout = 1) {
     };
 }
 
+export function openWorkTemplatesModal() {
```
Comment (mhd-sln) [inline actions/views/onboarding_tasks.ts:50]: any particular reason for using the plural and on import changing it to singular ?

Comment (JulienTant) [reply actions/views/onboarding_tasks.ts:50]: Yeah the reason is that my IDE sucks at refactoring LOL changing it all to singular, thanks @mhd-sln

File: actions/views/onboarding_tasks.ts [hunk 2]
```diff
@@ -45,6 +47,21 @@ export function openInvitationsModal(timeout = 1) {
     };
 }
 
+export function openWorkTemplateModal() {
+    return (dispatch: DispatchFunc) => {
+        dispatch(switchToChannels());
```
Comment (mhd-sln) [inline actions/views/onboarding_tasks.ts:52]:
```
dispatch(switchToChannels())
  .then(()=> {
    dispatch(openModal({....}))
})
```
maybe ?

**Review by mhd-sln — APPROVED** [review]

**Review by tiagodll — APPROVED** [review]
code LGTM

I would not put the templates in the top of the list, since the users probably have to learn more about the product before diving into the templates

but that is probably a design choice, nothing regarding this PR

**Comment by JulienTant** [comment]
@tiagodll  That was a bit unexpected for me too but the real goal of work templates from a growth perspective is to help user see asap what makes us different than slack: we're not just a messaging app. So I guess if you look at it through this lens, it kinda makes sense why we have this first, you get a hands first experience of having channels + boards + playbooks all linked together.

**Pull request merged**